alf/examples/merlin_dmlab_collect_good_objects.gin

```diff
@@ -0,0 +1,77 @@
+# MERLIN on deepmindlab rooms_collect_good_objects
+
+import alf.algorithms.merlin_algorithm
+import alf.algorithms.decoding_algorithm
+import alf.trainers.on_policy_trainer
+
+# `DeepmindLab` are required,
+#   see `https://github.com/deepmind/lab` to build `DeepmindLab`
+
+# environment config
+import alf.environments.suite_dmlab
+import alf.environments.wrappers
+
+# environment config
+create_environment.env_load_fn=@suite_dmlab.load
+create_environment.env_name="contributed/dmlab30/rooms_collect_good_objects_train"
+create_environment.num_parallel_environments=16
+suite_dmlab.load.wrap_with_process=0
+suite_dmlab.load.gym_env_wrappers=(@FrameResize, )
+FrameResize.width=64
+FrameResize.height=64
+
+suite_dmlab.action_discretize.jump=()
+suite_dmlab.action_discretize.crouch=()
+suite_dmlab.action_discretize.look_down_up_pixels_per_frame=()
+suite_dmlab.action_discretize.look_left_right_pixels_per_frame=(-30, 30)
+
+# build deepmind_lab with  --define graphics=osmesa_or_glx
+suite_dmlab.DeepmindLabEnv.renderer='hardware'
+
+observation_spec=@get_observation_spec()
+action_spec=@get_action_spec()
+
+# algorithm config
+LATENT_DIM=200
+
+img_enc/ResnetEncodingNetwork.input_tensor_spec=%observation_spec
+
+latent/TensorSpec.shape=(%LATENT_DIM,)
+img_dec/ResnetDecodingNetwork.input_tensor_spec=@latent/TensorSpec()
+img_dec/DecodingAlgorithm.loss_weight=100
+img_dec/DecodingAlgorithm.decoder=@img_dec/ResnetDecodingNetwork()
+
+MerlinAlgorithm.encoders=@img_enc/ResnetEncodingNetwork()
+MerlinAlgorithm.decoders=@img_dec/DecodingAlgorithm()
+MerlinAlgorithm.latent_dim=%LATENT_DIM
+MerlinAlgorithm.lstm_size=(256, 256)
+MerlinAlgorithm.memory_size=1350
+
+# ActorCriticLoss.gamma = 0.96
+# ActorCriticLoss.use_gae=True
+# ActorCriticLoss.use_td_lambda_return=True
+# ActorCriticLoss.td_lambda=0.9
+# ActorCriticLoss.entropy_regularization=0.01
```

Comment:
do you need these parameters?

Reply:
these are default parameters mentioned in the paper , i did not use them in this experiment (it has better performance when not use them)

Reply:
Then please make a comment about this.